packages/fxa-settings/src/index.tsx

```diff
@@ -39,6 +41,27 @@ try {
     return document.head.querySelector(name);
   });
 
+  // We do a best effort recovery of the user's signed in state
+  // There is no guarantee the event will be triggered before we
+  // have initialized our GraphQL client, because that depends on the
+  // browser responding to the web channel message quickly enough
+  firefox.addEventListener(FirefoxCommand.FxAStatus, (event) => {
```

Comment:
This may not make sense in a non-firefox context so we might want to branch out based on whether the context is a firefox browser or not